tests/upgrades/test_role.py

```diff
@@ -82,6 +88,8 @@ class TestBuiltInRolesLocked:
 
         1. Builtin roles of satellite should be locked and non-editable
         2. Built in roles of satellite should be allowed to clone
+
+    :CaseAutomation: NotAutomated
```

Comment:
If you already have class marked as stubbed, along with `:CaseAutomation: NotAutomated`, so you can ignore adding `CaseAutomation` at tests level for pre and post scenarios, it is redundant.

Reply:
Done